tests/integration/examples/test_notebooks_gpu.py

```diff
@@ -206,24 +206,6 @@ def test_xdeepfm_integration(
         )
 
 
-# TODO: remove tf dependency in this notebook and drop gpu marker
```

Comment:
Why remove the test? We could use an "xlearn" marker to disable it from the pipeline, and maybe add a separate pipeline in the future.

Reply:
On a DSVM, xlearn should be pip installable, because cmake is already installed at `/usr/local/bin/cmake`. So the test would run to completion, wouldn't it?